package.json

```diff
@@ -25,6 +25,7 @@
     "grunt-jsonlint": "^1.0.4",
     "grunt-jsxhint": "^0.6.0",
     "grunt-lintspaces": "^0.7.0",
+    "node-lintspaces": "shaoshing/node-lintspaces",
```

Comment:
Is this extra package actually being used? Does `grunt-lintspaces`somehow pick it up?